test/run_envoy_tests.sh

```diff
@@ -61,10 +61,16 @@ then
   exit 0
 fi
 
+# TODO(htuch): Clean this up when Bazelifying the hot restart test below.
+HOT_RESTART_JSON="$TEST_SRCDIR"/test/config/integration/hot_restart.json
+cat "$TEST_TMPDIR"/test/config/integration/server.json |
```

Comment:
I don't see how the flag would work unless it somehow talked to the existing process over RPC.

Actually, think the shared memory stuff might also be broken for bazel? How does that work? It definitely won't allow concurrently running the main server.

Let's just discuss in #654.